WordPress/pattern-directory
# Pattern Stats: Add CSV export for snapshots

Adds a form on the WP Admin stats screen for exporting a date range of stats snapshots to a CSV file.

Brings over the `Export_CSV` class from the WordCamp codebase.


### Screenshots

![snapshot-export](https://user-images.githubusercontent.com/916023/126581648-82403ce1-e740-41c0-8867-8edf52860db4.jpg)


### How to test the changes in this Pull Request:

1. Load this PR onto your sandbox so you can try exporting real data.
2. Go to the Stats screen, select a date range, and click the Export button.
3. Check the resulting spreadsheet to make sure the data looks legit.
4. See if you can get the form to spit out an `error.txt` file instead of a `.csv` file 😄
